contrib/repairStatus.py

```diff
@@ -0,0 +1,199 @@
+from __future__ import print_function
+import json
+import argparse
+import requests
+import re
+import logging as log
+from datetime import datetime, timedelta
+
+
+def generate_range_repair_query (node, start_time, current_time):
+  query_json = {
```

Comment:
Definitely go with a 4-space indent :).  "pip install pep8" can tell you about other style-guide stuff.